files/package.json

```diff
@@ -5,24 +5,20 @@
   "license": "MIT",
   "author": "",
   "scripts": {
-    "prepare": "yarn workspace <%= addonName %> run prepare",
-    "start": "npm-run-all --parallel start:*",
+    "prepare": "yarn build",
+    "build": "yarn workspace <%= addonName %> run build",
+
+    "start": "concurrently 'npm:start:*' --restart-after 5000 --prefix-colors cyan,white,yellow",
     "start:addon": "yarn workspace <%= addonName %> run start",
-    "start:test-app": "yarn workspace test-app run start",
-    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
-    "lint:addon": "yarn workspace <%= addonName %> run lint",
-    "lint:test-app": "yarn workspace test-app run lint",
-    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:fix:*",
-    "lint:fix:addon": "yarn workspace <%= addonName %> run lint:fix",
-    "lint:fix:test-app": "yarn workspace test-app run lint:fix",
-    "test": "npm-run-all --aggregate-output --continue-on-error --parallel \"test:!(watch)\"",
-    "test:watch": "npm-run-all --aggregate-output --continue-on-error --parallel test:watch:*",
-    "test:test-app": "yarn workspace test-app run test",
-    "test:watch:test-app": "yarn workspace test-app run test:watch",
-    "test:watch:addon": "yarn workspace <%= addonName %> run start"
+    "start:test": "yarn workspace <%= testAppInfo.name.dashed %> run start",
+
+    "lint": "yarn workspaces run lint",
```

Comment:
lint commands are now delegated to each package

Reply:
yeah, I'm a fan of easier overall -- trying to optimize stuff like this has always seemed heavily dependent on the environment to me.  Like, a lot of times sub-commands will do their own parallelization, and when you have multiple parallel things running, unaware of eachother, you can sometimes have _slower_ results, because you don't have enough CPUs -- as maybe each command is using `numThreads - 2` or something like that -- if you have a 16 thread CPU, you're now 12 threads over, causing a bunch of queueing.

I also kind of what to be as similar as possible between all the package managers.

So, like, for pnpm, I have this (and I don't know if this invalidates everything I said above about parallelizing, because pnpm seesm to do parallel by default):
```
❯ time pnpm lint

> limber-monorepo@ lint /home/nullvoxpopuli/Development/NullVoxPopuli/limber
> pnpm --filter='*' lint

Scope: 7 of 8 workspace projects
packages/consts lint$ concurrently 'npm:lint:*(!fix)' --names 'lint:'
│ [lint:js] 
│ [lint:js] > @nullvoxpopuli/limber-consts@0.0.0 lint:js
│ [lint:js] > eslint . --cache
│ [lint:js] 
│ [lint:js] npm run lint:js exited with code 0
└─ Done in 826ms
packages/horizon-theme lint$ concurrently 'npm:lint:*(!fix)' --names 'lint:'
│ [lint:js] 
│ [lint:js] > @nullvoxpopuli/horizon-theme@0.0.0 lint:js
│ [lint:js] > eslint . --cache
│ [lint:js] 
│ [lint:js] npm run lint:js exited with code 0
└─ Done in 866ms
packages/transpilation lint$ concurrently 'npm:lint:*(!fix)' --names 'lint:'
│ [lint:js] 
│ [lint:js] > @nullvoxpopuli/limber-transpilation@0.0.0 lint:js
│ [lint:js] > eslint . --cache
│ [lint:js] 
│ [lint:js] npm run lint:js exited with code 0
└─ Done in 1.5s
packages/codemirror lint$ concurrently 'npm:lint:*(!fix)' --names 'lint:'
│ [lint:js] 
│ [lint:js] > @nullvoxpopuli/limber-codemirror@0.0.0 lint:js
│ [lint:js] > eslint . --cache
│ [lint:js] 
│ [lint:js] npm run lint:js exited with code 0
└─ Done in 1.2s
packages/monaco lint$ concurrently 'npm:lint:*(!fix)' --names 'lint:'
│ [lint:js] 
│ [lint:js] > @nullvoxpopuli/limber-monaco@0.0.0 lint:js
│ [lint:js] > eslint . --cache
│ [lint:js] 
│ [lint:js] npm run lint:js exited with code 0
└─ Done in 1.2s
frontend lint$ concurrently 'npm:lint:*(!fix)' --names 'lint:'
[1 lines collapsed]
│ [lint:hbs] > limber@0.0.0 lint:hbs
│ [lint:hbs] > ember-template-lint . --no-error-on-unmatched-pattern
│ [lint:hbs] 
│ [lint:js] 
│ [lint:js] > limber@0.0.0 lint:js
│ [lint:js] > eslint . --cache
│ [lint:js] 
│ [lint:hbs] 
│ [lint:hbs] npm run lint:hbs exited with code 0
│ [lint:js] npm run lint:js exited with code 0
└─ Done in 1.4s

real	0m4.910s
user	0m9.359s
sys	0m1.223s
```

which gives some nice color-coded output, too

![image](https://user-images.githubusercontent.com/199018/177203625-9bacce53-5823-4fb5-b192-a0ce245226e5.png)

Reply:
I see your point, and it makes sense. So yeah, the simpler setup then probably makes more sense!